static/css/base/helpers-common.less

```diff
@@ -44,3 +44,10 @@ a.hoverlink {
   text-decoration: underline;
   text-decoration-style: dotted;
 }
+
+.local-date-time {
+  color: @dark-green;
+  font-size: .7em;
```

Comment:
It may be best to use an exact size here.  `.7em` is very small on the My Books page:
![image](https://user-images.githubusercontent.com/28732543/208810441-529737c5-ab10-4f89-824d-d99bde4e00b1.png)

It looks good on the books page, though...